<launch>
    <node pkg="coax_control" type="coax_taxing" name="coax_taxing" output="screen"/>
        
        <param name="neutral_roll" type="double" value="-0.0106"/>
        <param name="neutral_pitch" type="double" value="-0.02129"/>
        <param name="droll" type="double" value="0"/>
        <param name="dpitch" type="double" value="0.03490"/>

        <param name="Kp_ori_phi" type="double" value="1.0"/>        
        <param name="Kd_ori_phi" type="double" value="0.5"/>
</launch>